acmwebsite/controllers/root.py

```diff
@@ -47,10 +48,11 @@ class RootController(BaseController):
     mailinglist = MailingListController()
     u = UsersController()
     m = MeetingsController()
-    s = SurveysController()
+    survey = SurveysController()
```

Comment:
What's up with this change? Remember, the `SurveysController` looks up a survey by ID, under `/s/#`. There is code elsewhere that relies on this endpoint existing.